framework/src/main/java/org/checkerframework/framework/flow/CFAbstractValue.java

```diff
@@ -51,9 +55,21 @@
     /** The analysis class this value belongs to. */
     protected final CFAbstractAnalysis<V, ?, ?> analysis;
 
+    /** The underlying (Java) type in this abstract value. */
     protected final TypeMirror underlyingType;
+    /** The annotations in this abstract value. */
     protected final Set<AnnotationMirror> annotations;
 
+    /** Formats annotations without their package names. */
+    AnnotationFormatter annoFormatter = new DefaultAnnotationFormatter();
```

Comment:
I'm a bit concerned about adding this field. We create many abstract values and this will make that more costly.
We only call `toString`/`toStringSimple` when we debug/visualize the CFG, so it would overall probably be much better to only create the formatter in `toStringSimple`.
What do you think?

Reply:
I made the field static.  It has no state, but since its methods are non-static, there must be an instance of AnnotationFormatter.  But that instance can be the same for all uses.